### PR TITLE
feat: added alert and button components

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Added Alert and Button Component

## Description
Created the following components

1. Alert
  - Standard Alerts
  - Alerts With Icon
  - Alerts with close button
 
2. Button
  - Solid Buttons
  - Outlined Buttons
  - Buttons with Icons
  - Floating Action Buttons
  
## Output 
image
image

### CODE REVIEW

**Review by pnchinmay — APPROVED** [review]

File: pages/button.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        </p>
+        <section class="component__display">
+          <a href="#" class="btn btn--primary" role="button"
+            >Solid Link Button</a
+          >
+          <a href="#" class="btn btn--outline-primary" role="button"
+            >Outlined Link Button</a
+          >
+          <a href="#" class="btn btn--link-text" role="button"
+            >Solid Link Button</a
```
Comment (manujagobind) [inline pages/button.html]: Content here should be Text Button

File: pages/button.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+          >
+        </section>
+        <pre class="component__code-display"><code class="language-html">
+    &lt;a class="btn btn--primary" role="button" href="#"&gt;Solid Link Button&lt;/a&gt;
+    
+    &lt;a class="btn btn--outline-primary" role="button" href="#"&gt;Outlined Link Button&lt;/a&gt;
+
+    &lt;a href="#" class="btn btn--link-text" role="button"&gt;Solid Link Button&lt;/a&gt;
```
Comment (manujagobind) [inline pages/button.html]: Content here should be Text Button

**Review by manujagobind — COMMENTED** [review]
Good work!

**Review by manujagobind — APPROVED** [review]

File: pages/button.html (hunk 1 above)
Comment (srv-pranjal) [reply pages/button.html]: Fixed

File: pages/button.html (hunk 2 above)
Comment (srv-pranjal) [reply pages/button.html]: Fixed

**Pull request merged**